CommunityToolShedMvc/Views/Community/Index.cshtml

```diff
@@ -9,21 +9,27 @@
 
 <h3>Community Toolbox 🧰</h3>
 
-<div style="width: 30%">
+<div style="width: 40%">
```

Comment:
Not a big deal, but there are Bootstrap grid layout classes for constraining the width of your layout: https://getbootstrap.com/docs/4.3/layout/grid/#setting-one-column-width